src/System.Slices/src/System/Span.cs

```diff
@@ -335,7 +320,7 @@ public override bool Equals(object obj)
         /// gain: performance: no boundaries check (single operation) 
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private T GetItemWithoutBoundariesCheck(int index)
+        internal T GetItemWithoutBoundariesCheck(int index)
```

Comment:
Hmm, I really don't like this method being internal. It's just so easy to misuse it. What are the implications of not making it internal (and leaving it private)?